packages/api/test/repository-test.js

```diff
@@ -15,12 +15,20 @@
 const { describe, it, after, before } = require('mocha');
 const Repository = require('../src/repository');
 const assert = require('assert');
+const fetch = require('node-fetch');
```

Comment:
looks like this `fetch` dependency isn't used.

In practice, I'd try to keep use of `node-fetch` to the `server-test.js`, and have tests that run against models not perform HTTP requests.